### DISCUSSION TITLE
Your Unix terminal is ugly! Here's how to make it beautiful.

### DISCUSSION THREAD

**Discussion by giscus** [KyleTryon/techsquidtv.com#96] [Blog Comments]
# Your Unix terminal is ugly! Here's how to make it beautiful.

Make your Unix terminal beautiful and functional. Just because you live in the command line doesn't mean you have to live in the 80's

https://techsquidtv.com/blog/fixing_an_ugly_terminal

**Comment by zmarkan** [comment]
Great writeup!
I also like [Hyper](https://hyper.is/) as a terminal emulator - it's from the folks at Vercel, and similar to Tabby made with web technologies

**Comment by TroyGarseauh** [comment]
Going to take a look at Tabby and definitely start using ZSH. Thanks for the article was packed with lots of great info 🦑
